core/src/test/java/io/questdb/cutlass/http/ImportIODispatcherTest.java

```diff
@@ -263,7 +236,7 @@
             "|              1  |                                              Col2  |                   STRING  |           0  |\r\n" +
             "|              2  |                                              Col3  |                   STRING  |           0  |\r\n" +
             "|              3  |                                              Col4  |                   STRING  |           0  |\r\n" +
-            "|              4  |                                   Pickup_DateTime  |                TIMESTAMP  |           0  |\r\n" +
+            "|              4  |                                   Pickup_DateTime  |            [D] TIMESTAMP  |           0  |\r\n" +
```

Comment:
Designated already was marked in the row 3 of the first section here
```
Timestamp | Pickup_DateTime
```

Reply:
yeah, I will remove the funky `[D]`, I added it because for indexed symbols I add some "colour", but for designated ts this is noise :P